### ISSUE TITLE
Add an option to clear songs playback cache

### ISSUE THREAD

**Issue by Nikks84** [25huizengek1/ViTune#175]
If possible there should be an option so that the cache of songs playback & images is not stored, if not, there should be an option so that we can clear the cache of songs playback & images manually from settings > cache.

It's can be helpful to clear songs playback & images cache option inbuilt in settings > cache
That's it.

**Comment by 25huizengek1** [comment]
So you want to be able to clear both caches individually? You can already clear the cache in app info.

**Comment by Nikks84** [comment]
> So you want to be able to clear both caches individually? You can already clear the cache in app info.

Yes but too lengthy process i want this feature is inbuilt the app

**Comment by dtomvan** [comment]
Hmmm...

The current situation:
1. Click icon
2. App info
3. Clear cache

Your proposal:
1. Settings in app
2. Cache
3. Clear

**Comment by Nikks84** [comment]
> Hmmm...
> 
> The current situation:
> 
> 1. Click icon
> 2. App info
> 3. Clear cache
> 
> Your proposal:
> 
> 1. Settings in app
> 2. Cache
> 3. Clear

Yes

**Comment by 25huizengek1** [comment]
It's definitely not a lengthy process, but I get that some people want to clear image cache without destroying your offline cache. I'll try to add this after the refactoring/rebranding.

**Comment by 25huizengek1** [comment]
Due to the nature of how the file system works on Android, I unfortunately can't get this to behave consistently.

Closing for now.
